src/NUnitFramework/framework/Api/DefaultTestAssemblyBuilder.cs

```diff
@@ -148,22 +153,24 @@ private TestSuite Build(Assembly assembly, string assemblyPath, IDictionary<stri
                             }
                         }
                 }
+                
+                var fixtures = GetFixtures(assembly, GetValueOrDefault(options, FrameworkPackageSettings.LOAD) as IList);
```

Comment:
Consistency, instead of having multiple ways to do the same thing. No problem to change back.